src/app/project/comment-periods/add-edit-comment-period/add-edit-comment-period.component.ts

```diff
@@ -13,7 +13,7 @@ import { DocumentService } from 'app/services/document.service';
 import { StorageService } from 'app/services/storage.service';
 
 import { Utils } from 'app/shared/utils/utils';
-
+import {CommentPeriodBusiness} from '@bit/eao.epic.comment-period-business-logic';
```

Comment:
Will this reference the bit cloud or the code we download when we yarn install?

Reply:
This will reference the code we download when we yarn install, working as a dependency.
there is a .npmrc file sitting in the root of the project, that tells npm to install it as dependency. 

Please do not merge this and related pr before everyone agree that we will use bit in the feature.

Reply:
You should set this as a draft then so no one accidentally pull this in.